opensearch_py_ml/ml_models/sentencetransformermodel.py

```diff
@@ -1126,6 +1193,9 @@ def make_model_config_json(
             },
         }
 
+        if description is not None:
+            model_config_content["description"] = description
```

Comment:
If description is `None`, can we add some general description? For example "This is a sentence transformer model which can convert text into embedding (float vector)"?

Reply:
Sure.